core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -64,7 +64,7 @@
             ng-mouseover="$ctrl.openSubmenu($event, 'classroomMenu')"
             ng-mouseleave="$ctrl.closeSubmenuIfNotMobile($event)"
             role="menu">
-          <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'classroomMenu', {shiftTab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link" href="/classroom/Math" translate="I18N_TOPNAV_CLASSROOM_BASIC_MATHS"></a></li>
+          <li class="nav-item"> <a ng-keydown="$ctrl.onMenuKeypress($event, 'classroomMenu', {shiftTab: $ctrl.ACTION_CLOSE})" class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-classroom-link" href="/Math" translate="I18N_TOPNAV_CLASSROOM_BASIC_MATHS"></a></li>
```

Comment:
/Math or /math? Please check other cases, we should be consistent everywhere.

Also did you test this manually? **Please always test all your changes manually, using a local dev server, before asking for a review.**

Reply:
Discussion for capitalization in the comment [above](https://github.com/oppia/oppia/pull/9109#discussion_r410668710).

Yes, I've tested all changes thoroughly creating various scenarios. I've made sure that no console errors occur and that things work smoothly, before pushing my changes.

Reply:
Hm, I thought you mentioned in a comment that URLs were case-sensitive and that the capitalized link would have given you a 404. Was that not uncovered by manual testing before?

Reply:
Oh, I'm terribly sorry about that. This is actually hidden behind a flag and I forgot to toggle it while performing manual tests. I've changed it now and tested it, it works as expected.